src/dmd/func.d

```diff
@@ -1891,6 +1924,17 @@ extern (C++) class FuncDeclaration : Declaration
         return false;
     }
 
+    /****************************************************
+     * Check whether result variable can be built.
+     * Returns:
+     *     `true` if the function has a return type that
+     *     is different from `void`.
+     */
+    final bool canBuildResultVar(){
```

Comment:
I see that this is only used by the module its declared in. So make it private, and extern(D).